docs/ref/settings.txt

```diff
@@ -37,7 +37,7 @@ a model object and return its URL. This is a way of inserting or overriding
         'news.story': lambda o: "/stories/%s/%s/" % (o.pub_year, o.slug),
     }
 
-Note that the model name used in this setting should be all lower-case, regardless
+Note that the model name used in this setting should be all lowercase, regardless
```

Comment:
The spelling "lower-case" appears several times throughout the project. Should we normalize this spelling globally?

```
./docs/ref/models/fields.txt:1682:the ``OneToOneField``, Django will use the lower-case name of the current model
./docs/ref/settings.txt:40:Note that the model name used in this setting should be all lower-case, regardless
./docs/topics/db/models.txt:984:- ``'%(class)s'`` is replaced by the lower-cased name of the child class
./docs/topics/db/models.txt:986:- ``'%(app_label)s'`` is replaced by the lower-cased name of the app the child
./docs/topics/db/models.txt:1068:``Place`` object to the ``Restaurant`` object by using the lower-case version
./docs/topics/i18n/index.txt:77:      are generally represented in lower-case, but the HTTP ``Accept-Language``
./tests/many_to_one/models.py:47:# created (the field names being lower-cased versions of their opposite
./django/http/request.py:541:    Returned domain is lower-cased. If the host is invalid, the domain will be
./django/http/request.py:569:    Note: This function assumes that the given host is lower-cased and has
./django/http/response.py:39:        # _headers is a mapping of the lower-case name to the original case of
./django/contrib/gis/feeds.py:62:                # Getting the lower-case geometry type.
./django/contrib/gis/gdal/geomtype.py:29:    # Reverse type dictionary, keyed by lower-case of the name.
./django/db/models/fields/reverse_related.py:152:        # many-to-many objects. It uses the lower-cased object_name + "_set",
```

Let me know if you'd like to do this, otherwise I don't mind.

Reply:
I say we normal-ize it, it's worth doing in a separate pull-request

Reply:
Done in  #10440.